### DISCUSSION TITLE
Rename browser version "math" to "Mathjs"?

### DISCUSSION THREAD

**Discussion by manake** [josdejong/mathjs#2783] [Ideas]
If it's math.js then maybe obj should be `mathjs` instead of `math`? Not sure.

Could even be `Math.mathjs` or `MathJs` or `Mathjs` (probably best since the capital `M` would match that of `Math`).

**Comment by josdejong** [comment]
Yeah, maybe we could rename the instance in the examples and docs from `math` to `mathjs`. Renaming the browser bundle export will be a breaking changes though, so we need to be careful there. At this point it's not important enough for me to introduce such a breaking change. Since the JavaScript world is moving to ES modules, it becomes less important too, since your typically loading the library like `import { sqrt } from 'mathjs'` or something like that.
